Tests/pageObject/RadioNavigationalWarningsAdminListEndUser.page.ts

```diff
@@ -102,4 +119,27 @@ export default class RadioNavigationalWarningsListEndUser {
     }
     expect(compareDate).toBeTruthy();
   }
+
+  public async verifySelectOption() {
```

Comment:
This method appears to be doing a series of actions and checks.  Can you split this up?

Reply:
resolved